packages/fx-core/src/core/middleware/utils/migrationContext.ts

```diff
@@ -6,7 +6,7 @@ import fs, { CopyOptions, EnsureOptions, PathLike, WriteFileOptions } from "fs-e
 import path from "path";
 import { CoreHookContext } from "../../types";
 
-const teamsfxFolder = "teamsfx";
+export const teamsfxFolder = "teamsfx";
```

Comment:
there's `SettingsFolderName` in `@microsoft/teamsfx-api` package, you can import this in projectMigratorV3.ts. No need to export here.